Seem like Makefile run command not correct

I just got the error when i run `make run`.   

```shell
...
kustomize build manifests/setup | sed -e '/creationTimestamp/d' > manifests/setup/setup.yaml
go run ./main.go
stat ./main.go: no such file or directory
make: *** [run] Error 1
```
May be `cmd/manager/main.go` not `./main.go` ? 

```shell
run: generate fmt vet manifests
	go run ./main.go
```

--- a/Makefile
+++ b/Makefile
@@ -25,7 +25,7 @@ manager: generate fmt vet
 
 # Run against the configured Kubernetes cluster in ~/.kube/config
 run: generate fmt vet manifests
-	go run ./main.go
+	go run cmd/manager/main.go
 
 # Install CRDs into a cluster
 install: manifests

